js/ui/grid_core/ui.grid_core.data_controller.js

```diff
@@ -91,6 +91,13 @@ module.exports = {
                             that.option('paging.' + optionName, value);
                             that._skipProcessingPagingChange = false;
 
+                            const editingController = that.getController('editing');
```

Comment:
1) Move this code to editing module
2) Check virtual scrolling because virtual scrolling uses paging